src/hooks/use-archive-site.ts

```diff
@@ -48,20 +49,22 @@ export function useArchiveSite() {
 								...snapshot,
 								isLoading: false,
 							} );
+							setIsSnapshotProcessing( false );
```

Comment:
I have set the progress to finish here instead of `finally{}` because this is actually where the snapshot is marked as active.

If we kept the logic in `finally{}`, the progress bar would get reset before the snapshot is ready.